incubator/logstash/templates/service-lb.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.serviceLoadBalancer.enabled }}
```

Comment:
Why do you need a second service? This seems to duplicate the existing one that can already be configured to be of type LoadBalancer if desired, which I think is better anyways.

Reply:
Yes. This is a good point and I agree. I've deleted the second service and configured the first service to accept annotations passed from values.yaml.   Thanks!